db/seeds.rb

```diff
@@ -130,7 +130,8 @@
     'application deadline' => Date.parse('20230831'),
     'student count' => 17_192,
     dorm: true,
-    'email domain' => '@siswa.um.edu.my'
+    'email domain' => '@siswa.um.edu.my',
```

Comment:
remove the last comma if email domain is the last item

Reply:
Fixed in latest push